info/README.md

```diff
@@ -0,0 +1,7 @@
+UPDATE MAIN README
```

Comment:
Please make it an H1, and don't forget the article:

```markdown
# UPDATE THE MAIN README
```

Reply:
Done [78342c5](https://github.com/saveourtool/diktat/pull/1384/commits/78342c53dd92a42802f781323a6cc10b03605118)

Please check it again